src/components/ToDo/ToDoList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import LastRow from './LastRow'
+import ToDoCard from './ToDoCard'
+import { Todo } from '../../pages/FrontendMentor/Todo'
+import { Wrapper } from './styles/ToDoList.styled'
+
+type Props = {
+  todos: Todo[]
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>
+  isDarkMode: boolean
+}
+
+const ToDoList = (props: Props) => {
+  return (
+    <Wrapper>
+      {props.todos
+        .filter((todo: Todo) => {
+          return todo.isVisible
+        })
+        .map((todo: Todo) => (
+          <ToDoCard
```

Comment:
Perhaps its better to stick to ToDo and toDo since you use ToDoCard and ToDo for your style folder so that you won't be confused which to use in the future when the code is merged to main.